lua/catppuccin/groups/integrations/treesitter.lua

```diff
@@ -23,7 +23,9 @@ If you want to stay on nvim 0.7, either disable the integration or pin catppucci
 		["@constant.builtin"] = { fg = C.peach, style = O.styles.keywords or {} }, -- For constant that are built in the language: nil in Lua.
 		["@constant.macro"] = { link = "Macro" }, -- For constants that are defined by macros: NULL in C.
 
-		["@module"] = { fg = C.lavender, style = { "italic" } }, -- For identifiers referring to modules and namespaces.
+		-- This is difficult to avoid hardcoding, as there is no user option for this (yet).
+		-- For now, binding to `conditionals` because it is italic by default.
+		["@module"] = { fg = C.lavender, style = O.styles.conditionals or { "italic" } }, -- For identifiers referring to modules and namespaces.
```

Comment:
This isn't a conditional though?. You can always just `style = {}` in userconfig or is it not

Reply:
This precisely is the problem; the user’s config does not override this setting. That’s what I changed, by binding the setting for `@module` to the users setting for conditionals (which, by default, is also italic). This way the user can change if modules are italic or not, while the default of italic is still preserved.